### ISSUE TITLE
Moving Creating an Archive Index and Creating a Search Pages to Theme Handbook

### ISSUE THREAD

**Issue by TeBenachi** [WordPress/Documentation-Issue-Tracker#381]
I think the [Creating an Archive Index](https://codex.wordpress.org/Creating_an_Archive_Index) can be added under the [Template Files Section](https://developer.wordpress.org/themes/template-files-section/) on the Theme Handbook. 

Draft => https://developer.wordpress.org/wp-admin/post.php?post=140566&action=edit

And same for the [Creating a Search Page](https://codex.wordpress.org/Creating_a_Search_Page). 

Draft =>  https://developer.wordpress.org/wp-admin/post.php?post=140561&action=edit

**Comment by carolinan** [comment]
I will review this update this week.

**Comment by carolinan** [comment]
I don't think we should copy the text as is from the old codex pages because they have a different tone than most other pages in the handbook, and they are lengthy, not to the point.

**Comment by carolinan** [comment]
I found that it overlaps with https://developer.wordpress.org/themes/template-files-section/taxonomy-templates/
But, I also found it confusing that the archive.php template is not more prominent on that page.

**Comment by carolinan** [comment]
I think the content of the "search page" is out of date. And I find the introduction, the first paragraphs, very confusing.
I think it needs a complete rewrite.
I don't believe that creating a custom **page** template that you assign to a page in the block editor is the best practice for including a search results page.
But I would like a second pair of eyes before considering a complete rewrite.
@DaisyOlsen @justintadlock

**Comment by DaisyOlsen** [comment]
Regarding the search page implementation method. I agree that a custom page template seems the wrong approach to recommend since it should be `search` per the t[emplate hierarchy](https://developer.wordpress.org/themes/basics/template-hierarchy/#search-result) to integrate with core functionality fully.

That codex page has the feeling of being a tutorial more than a reference. A rewrite to be more concise for the handbook and promote the idea for the tutorial over to Learn.WordPress could be one possible path.
